spec/system/candidate_interface/entering_details/candidate_edits_or_deletes_volunteering_section_after_competing_the_section_spec.rb

```diff
@@ -4,6 +4,8 @@
   include CandidateHelper
 
   scenario 'Candidate adds or deletes a role after completing the section' do
+    FeatureFlag.deactivate('restructured_work_history')
```

Comment:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/4213/files#r592392701